src/index.ts

```diff
@@ -68,5 +73,7 @@ function removeNullOrUndef(params: ParamMap) {
 }
 
 function notNullOrUndefined(v: any) {
+  if (v === null || v === undefined) throw new TypeError(`Path parameter ${v} is not Valid`);
+
```

Comment:
Could you please create a separate PR for #12? I have some comments related to those changes; this way I could merge this PR more quickly and your work would show up as two separate contributions :)